app/js/results/measurementSensitivityDirective.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+define([],
+  function() {
+    var dependencies = [
+      'DeterministicResultsService'
+    ];
+    var MeasurementSensitivityDirective = function(
+      DeterministicResultsService
+    ) {
+      return {
+        restrict: 'E',
+        scope: {
+          aggregateState: '=',
+          alternatives: '=',
+          criteria: '=',
+          editMode: '=',
+          scenario: '='
+        },
+        templateUrl: './measurementSensitivityDirective.html',
+        link: function(scope) {
+          scope.doMeasurementSensitivity = doMeasurementSensitivity;
+
+          init();
+
+          scope.$on('elicit.legendChanged', function() {
+            init();
+          });
+
+          function init() {
+            scope.measurementsAlternative = scope.alternatives[0];
+            scope.measurementsCriterion = scope.criteria[0];
+            doMeasurementSensitivity();
+          }
+
+          function doMeasurementSensitivity() {
+            delete scope.measurementValues;
+            DeterministicResultsService.getMeasurementSensitivityResults(scope, scope.aggregateState.dePercentified).resultsPromise.then(function(result) {
+              scope.measurementValues = DeterministicResultsService.pataviResultToLineValues(result.results, scope.alternatives, scope.scenario.state.legend);
+
+              if (usePercentage(scope.measurementsCriterion.dataSources[0])) {
```

Comment:
I'm confused how this intersects with the workspacesettings.usepercentage (because here only one datasource is checked?)